From 37c92c53b272a60dce1d7aa7f187944e8dcaaed4 Mon Sep 17 00:00:00 2001
From: Aditya Kumar Singh <quic_adisi@quicinc.com>
Date: Thu, 17 Nov 2022 22:09:07 +0530
Subject: [PATCH] ath12k: remove mac Tx management debug prints

A few debug prints were printed for every management Tx hapenning
in the driver. This leads to flooding of debug prints on console
when the debug logs are enabled.

Remove debug prints for Tx management packets.

Signed-off-by: Aditya Kumar Singh <quic_adisi@quicinc.com>
---
 drivers/net/wireless/ath/ath12k/mac.c | 7 +------
 drivers/net/wireless/ath/ath12k/wmi.c | 4 ----
 2 files changed, 1 insertion(+), 10 deletions(-)

--- a/drivers/net/wireless/ath/ath12k/mac.c
+++ b/drivers/net/wireless/ath/ath12k/mac.c
@@ -7517,8 +7517,7 @@ static int ath12k_mac_mgmt_tx_wmi(struct
 	buf_id = idr_alloc(&ar->txmgmt_idr, skb, 0,
 			   ATH12K_TX_MGMT_NUM_PENDING_MAX, GFP_ATOMIC);
 	spin_unlock_bh(&ar->txmgmt_idr_lock);
-	ath12k_dbg(ar->ab, ATH12K_DBG_MAC,
-		   "mac tx mgmt frame, buf id %d\n", buf_id);
+
 	if (buf_id < 0)
 		return -ENOSPC;
 
@@ -7597,7 +7596,7 @@ static void ath12k_mgmt_over_wmi_tx_work
 					    arvif->vdev_id, ret);
 				ath12k_mgmt_over_wmi_tx_drop(ar, skb);
 			} else {
-				ath12k_dbg(ar->ab, ATH12K_DBG_MAC,
+				ath12k_dbg(ar->ab, ATH12K_DBG_MGMT,
 					   "mac tx mgmt frame, vdev_id %d\n",
 					   arvif->vdev_id);
 			}
--- a/drivers/net/wireless/ath/ath12k/wmi.c
+++ b/drivers/net/wireless/ath/ath12k/wmi.c
@@ -5913,10 +5913,6 @@ skip_mgmt_stats:
 	if (num_mgmt < 0)
 		WARN_ON_ONCE(1);
 
-	ath12k_dbg(ar->ab, ATH12K_DBG_WMI,
-		   "wmi mgmt tx comp pending %d desc id %d\n",
-		   num_mgmt, desc_id);
-
 	if (!num_mgmt)
 		wake_up(&ar->txmgmt_empty_waitq);	
 
